installation/eks.md

```diff
@@ -0,0 +1,79 @@
+# Installation using Helmfile
+
+Installing Helm charts with lots of dependencies and CRDs is challenging; these instructions use Helmfile to mitigate issues with Helm. 
+
+This documentation focuses on installing the FinOps Stack in EKS clusters.
+
+## Pre-requisites
+
+- A EKS cluster with:
+   - kubectl access 
+   - (Optional) If your cluster has Spot Instances, EKS Pod Identities need to be configured. See [documentation](https://www.opencost.io/docs/configuration/aws#eks-pod-identities).
+- [Helmfile](https://helmfile.readthedocs.io/en/latest/#installation) installed on your local machine
+- Unless you want to access the Grafana dashboard via `kubectl port-forward` you'll need a domain name or external public  IP.
+
+## Installation
+
+### Configuration changes for your cluster environment
+
+1. To control which Finops Stack components to install, edit the [enabled.yaml](./installation/config/common/enabled.yaml) file
+1. Copy `./env_eks.tmpl` to `./.env` and replace the env var values accordingly.
+
+### Install everything using Helmfile
+
+For the first run:
+
+```bash
+set -a; source .env; set +a; helmfile apply --file Helmfile_eks.yaml --interactive
+```
+
+NOTE: it will take several minutes for all workloads to install and start running. Helmfile does display its progress in the terminal. All workloads get installed into the `finops-stack` namespace so you can also view progress using `kubectl`.
+
+To speed up subsequent runs:
+
+```bash
+set -a; source .env; set +a; helmfile apply --file Helmfile_eks.yaml --interactive --skip-deps
+```
+
+## Optional: Configure ingress for Grafana
+
+### Pre-requisites
+
+Already have an FQDN setup and registered with a public IP, e.g. grafana.example.com
```

Comment:
Presumably this isn't actually needed, and you could edit `/etc/hosts` to point to a public IP?